04-trade-protocol.md

```diff
@@ -34,14 +34,40 @@ A **trader** connects to the provider using the [secure transport defined in the
 * Service Interface
 
 ```protobuf
+syntax = "proto3";
+import "google/api/annotations.proto";
+
 service Trade {
-  rpc Markets(MarketsRequest) returns (MarketsReply);
-  rpc Balances(BalancesRequest) returns (BalancesReply);
-  rpc MarketPrice(MarketPriceRequest) returns (MarketPriceReply);
+  rpc Markets(MarketsRequest) returns (MarketsReply) {
+    option (google.api.http) = {
+      get: "/v1/markets"
+    };
+  }
+  rpc Balances(BalancesRequest) returns (BalancesReply) {
+    option (google.api.http) = {
+      get: "/v1/market/{base_asset}/{quote_asset}/balance"
```

Comment:
this is different than market/price signature.

I think is easier to make this POST as well and pass market in the BODY payload